### ISSUE TITLE
+  инструкция к  CLOUD  (zhivits@gmail.com)

### ISSUE THREAD

**Issue by zhivik007** [sbprov2/SBProV2#266]
Доброго времени суток!
Уважаемая команда программистов!, где то ранее, прозвучала информация о том, что готовится подробный мануал по алгоритму..., уточните пожалуйста информацию!

**Comment by icesoye** [comment]
Ну как вариант можно посмотреть мануал к v1 - настройки те же.

**Comment by kichkin38** [comment]
согласен,но помойму алгоритмы маленько изменились.Или я в халявной не мог так настроить.Да и разрабы говорили что улучшат клауд.

**Comment by icesoye** [comment]
Да я и не говорил что это последняя инстанция, просто как временная замена
